sky/optimizer.py

```diff
@@ -1292,6 +1292,7 @@ def _fill_in_launchable_resources(
         # If clouds provide hints, store them for later printing.
         hints: Dict[clouds.Cloud, str] = {}
         for cloud in clouds_list:
+            # import ipdb; ipdb.set_trace()
```

Comment:
accidentally?

Reply:
woops, you caught me.

Reply:
see 82f402f91be46455515f60ec8f4fc05666b815e4

Reply:
see d34865dee22fd96f2ec290d41523f43edd69e129